src/cdk/drag-drop/drag.ts

```diff
@@ -270,7 +270,13 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
       const rootElement = this._rootElement = this._getRootElement();
       rootElement.addEventListener('mousedown', this._pointerDown, passiveEventListenerOptions);
       rootElement.addEventListener('touchstart', this._pointerDown, passiveEventListenerOptions);
-      toggleNativeDragInteractions(rootElement , false);
+
+      const shouldEnableDragInteractions = () => Boolean(this.getChildHandles().length);
+      toggleNativeDragInteractions(rootElement, shouldEnableDragInteractions());
+      this._handles.changes
+        .pipe(map(() => shouldEnableDragInteractions()), distinctUntilChanged())
+        .subscribe((enableInteractions: boolean) =>
+          toggleNativeDragInteractions(rootElement, enableInteractions));
```

Comment:
It would be a bit more concise like this:

```ts
this._handles.changes.pipe(startWith(null)).subscribe(()=> {
   toggleNativeDragInteractions(rootElement, this.getChildHandles().length > 0));
});
```

Also note that the `distinctUntilChanged` won't really do anything in this case since you always get a new `QueryList`.

Reply:
Done. Thanks for this.